src/Button.js

```diff
@@ -2,18 +2,25 @@
 import React from 'react';
 
 function resetProduct() {
-  document.getElementById('productLogo').src='dist/img/DPW_Skull_Logo_Mask.jpg';
+  	document.getElementById('productLogo').src='img/DPW_Skull_Logo_Mask.jpg';
+
+	document.getElementById("eyesBlue").checked = false;
+	document.getElementById("eyesGreen").checked = false;
+	document.getElementById("eyesGrey").checked = false;
+		
 }
 
 const Button = (props) => {
 	//list multiple props here
-	const { type = '',
-	onClick,
-	propThree,
-	propFour } = props
+	const { 
```

Comment:
This looks fine to me, although a few changes you should probably make. I think you can remove the `propThree` and `propFour` props as these aren't needed for your button and don't really do anything. This will need to be updated in the return statement for the button also, so you'll need to remove them:

`<button type={type} onClick={onClick}>Reset to original</button>`

Reply:
Thanks @justtoconfirm makes perfect sense. These two suggestions have been actioned, committed and pushed to origin for you to review.

Reply:
@dpwit you'll probably also want to update the` onClick` prop to remove the `resetProduct`:

`onClick`

And you'll also want to add a `children` prop in so you can change the button label:

`<button type={type} onClick={onClick}>{children}</button>`

Reply:
Cheers mate - already done